client/extensions/wp-super-cache/preload-tab.jsx

```diff
@@ -50,7 +50,7 @@ class PreloadTab extends Component {
 	};
 
 	componentWillReceiveProps( nextProps ) {
-		this.setState( { preloadRefresh: parseInt( nextProps.fields.preload_interval, 10 ) === 0 ? false : true } );
+		this.setState( { preloadRefresh: parseInt( nextProps.fields.preload_interval, 10 ) !== 0 ? true : false } );
```

Comment:
Wow! 😆  Well, at least you know you can always count on me for your comic relief, whether intentional or not! Fixed in https://github.com/Automattic/wp-calypso/pull/14518/commits/256401083dc1c26e6547480cd2cf683a15c801bb.